nautobot/extras/datasources/git.py

```diff
@@ -271,34 +216,29 @@ def update_git_config_contexts(repository_record, job_result):
     for file_name in os.listdir(config_context_path):
         if not os.path.isfile(os.path.join(config_context_path, file_name)):
             continue
-        job_result.log(
-            f"Loading config context from `{file_name}`",
-            grouping="config contexts",
-            logger=logger,
-        )
+        msg = f"Loading config context from `{file_name}`"
+        logger.info(msg)
+        job_result.log(msg, grouping="config contexts")
```

Comment:
Do we need the double logging here? The same question throughout.

Reply:
The first line logs to the console, the second line logs to the database as a `JobLogEntry`